docs/how-to/how-to-run-docker-with-kata.md

```diff
@@ -24,7 +24,7 @@ You can learn more about about Docker-in-Docker at the following links:
 While normally DinD refers to running `docker` from inside a Docker container,
 Kata Containers 2.x allows only [supported runtimes][kata-2.x-supported-runtimes] (such as [`containerd`](../install/container-manager/containerd/containerd-install.md)).
 
-Running `docker` in a Kata Container implies creating Docker containers from inside a container managed by `containerd` (or another supported container manager), as illustrated below:
+Running `docker` in a Kata Containers implies creating Docker containers from inside a container managed by `containerd` (or another supported container manager), as illustrated below:
```

Comment:
Either "Running docker in a Kata Container" or "Running docker in Kata Containers"

Reply:
@cmaf I reverted these two `Containers` -> `Container`, please check again.